app/main/posts/views/card.html

```diff
@@ -3,7 +3,7 @@
     <div class="listing-item-select">
         <input ng-show="canSelect" type="checkbox" checklist-value="post.id" checklist-model="selectedPosts" ng-click="stopClickPropagation($event)">
     </div>
-    <div class="postcard-body" ng-click="clickAction($event)">
+    <div class="postcard-body" tabindex="0" role="button" ng-keypress="clickAction($event)" ng-click="clickAction($event)">
```

Comment:
@Angamanga  @rowasc, I was able to make some changes using angularJS ng-keypress attribute to make the div element clickable

|   Proposed changes   | 
| --------------------------------- |
|  ![post-onkeypress](https://user-images.githubusercontent.com/66112760/97757200-90e5eb80-1afc-11eb-8d06-9bdbd3a63fe5.gif)  |  

Please help me review will appreciate your feedback!